design/ui/pagination/src/Pagination.js

```diff
@@ -0,0 +1,292 @@
+import React, { Fragment } from 'react';
+import pt from 'prop-types';
+import classNames from 'classnames/bind';
+import { noop } from 'lodash';
+
+import { Button, Select, Header, Popup } from 'semantic-ui-react';
+import 'semantic-ui-css-custom';
+import styles from './Pagination.module.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /** Can render as a different tag or component */
+  as: pt.oneOfType([pt.string, pt.node]),
+  /** the items collection length */
+  count: pt.number,
+  /** entity name in plural form */
+  entityNamePlural: pt.string,
+  /** entity name in singular form */
+  entityNameSingular: pt.string,
+  /** specifies if the data is currently loading */
+  isLoading: pt.bool,
+  /** items per page limit */
+  limit: pt.number,
```

Comment:
`limit` also feels like I do not know what it is for without looking at the note about being for the page.